DEVLOPING_WITH_DOCKER.md

```diff
@@ -0,0 +1,26 @@
+# Developing with Docker
+
+Docker is a powerful tool that can be used to quickly spin up an OpenSearch cluster. When you follow the steps to run [OpenSearch with Docker](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/), you will find the Security Plugin already included in the basic distribution. 
+
+- [Developing with Docker](#developing-with-docker)
+    - [Configuring Security](#configuring-security)
+    - [Mounting Local Volumes](#mounting-local-volumes)
+    
+
+## Configuring Security 
+
+By default, the Docker installation of OpenSearch does not enable the Security plugin. In order to enable Security development, you will need set `DISABLE_SECURITY_PLUGIN=false`. This will install the demo certificates, and allow you to develop with realistic Security configurations. 
```

Comment:
Not only `DISABLE_SECURITY_PLUGIN`. `DISABLE_INSTALL_DEMO_CONFIG` and `DISABLE_SECURITY_DASHBOARDS_PLUGIN` too and from all services in the sample docker-compose file.

It may make sense to put a sample docker-compose with security here and leave the version as a placeholder to prevent it from becoming stale too quickly. Another option is to make a PR against the documentation website to provide a dev option with security as well.